### PR TITLE
some pep8 fixes in sage/all.py

### PR DESCRIPTION
**Pull request by fchapoton**
This is fixing

E272 multiple spaces before keyword

in all.py

### :memo: Checklist

- [x] The title is concise and informative.
- [x] The description explains in detail what this PR is about.
- [ ] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation and checked the documentation preview.

### CODE REVIEW

File: src/sage/all.py [hunk 1]
```diff
@@ -191,20 +193,27 @@
...
 register_unpickle_override('sage.libs.pari.gen_py', 'pari', pari)
 
 # Cache the contents of star imports.
 sage.misc.lazy_import.save_cache_file()
 
 
-### Debugging for Singular, see trac #10903
+# Debugging for Singular, see trac #10903
 # from sage.libs.singular.ring import poison_currRing
 # sys.settrace(poison_currRing)
```
Comment (kwankyu) [inline src/sage/all.py:221]: The first line is a comment to the following commented code lines.

Comment (kwankyu) [reply src/sage/all.py:221]:
```
# # Debugging for Singular, see Issue #10903
```
?

Comment (kwankyu) [reply src/sage/all.py:221]: it is capitalized "Issue"... How about "##" instead of "####"?

Comment (fchapoton) [reply src/sage/all.py:221]: come on, please spare my time and yours.

Comment (kwankyu) [reply src/sage/all.py:221]: OK. Not a big issue.

**Review by kwankyu — APPROVED** [review]
LGTM (except the issue :-)

**Pull request merged**